proposed/http-client/http-client.md

```diff
@@ -43,10 +43,9 @@ interface HttpClient
     /**
      * Sends a PSR-7 request.
      *
-     * If a request is sent without any prior configuration, an exception MUST NOT be thrown
-     * when a response is recieved, no matter the HTTP status code.
+     * An exception MUST NOT be thrown when a response is recieved, no matter the HTTP status code.
```

Comment:
This should probably contain: `when a valid response is received`.

Reply:
invalid checksum probably as well. can we rephrase this to be clear? lets insert "valid Response" - imo together with "no matter the HTTP status code" it should be clear that a 4xx or 5xx is not allowed to trigger an exception..